systems/common/events/event.go

```diff
@@ -274,15 +274,15 @@ var EventToEventConfig = map[EventId]EventConfig{
 		Title:       "Mesh Node Online",
 		Description: "Mesh Node Online",
 		Scope:       notif.SCOPE_ORG,
-		Type:        TypeDefault,
+		Type:        notif.TYPE_ACTIONABLE_INFO,
 	},
 	EventMeshNodeOffline: {
 		Key:         EventMeshNodeOffline,
 		Name:        "EventMeshNodeOffline",
 		Title:       "Mesh Node Offline",
 		Description: "Mesh Node Offline",
 		Scope:       notif.SCOPE_ORG,
-		Type:        TypeDefault,
+		Type:        notif.TYPE_ACTIONABLE_INFO,
```

Comment:
Why is "node offline" something we need to act on? I don't think it needs any action. It should just be TYPE INFO .

Reply:
This kind of notification supposed to lead to detail page. So in this case, if `ABC` node is offline that an actionable info which will lead to `ABC` notification detail page where use can perform curtain type of actions. like `restart`.